react/bulkImport/messages.ts

```diff
@@ -1,8 +1,18 @@
 import { defineMessages } from 'react-intl'
-import type { UploadMessageKey } from '@vtex/bulk-import-ui/dist/i18n/keys'
+import type {
+  ImportMessageKey,
+  UploadMessageKey,
+} from '@vtex/bulk-import-ui/dist/i18n/keys'
+
+export type BulkImportMessageKey =
+  | UploadMessageKey
+  | ImportMessageKey
+  | 'reportInformationListTotal'
+  | 'reportInformationImportSuccessFully'
+  | 'reportInformationImportFailed'
```

Comment:
What are these 3 for if they're not part of the expected keys from the bulk-import-ui library? I don't see them being used in this PR though.

Reply:
These keys are not part of bulk-import-ui, they are part of the application's functionalities, they were added in older PR's, but only in this one I noticed that they were missing in the typings